.github/workflows/analyze-target.yml

```diff
@@ -81,3 +81,13 @@ jobs:
         name: covscan
         path: |
           ./logs/*.err
+
+  result:
+    name: All tests are successful
+    if: ${{ always() }}
```

Comment:
Why do you set this line?

Reply:
So it also run on failure. If it would not run, it would not be present in the check list at all.